examples/lora/README.md

```diff
@@ -0,0 +1,323 @@
+# DreamBooth training example
+
+[DreamBooth](https://arxiv.org/abs/2208.12242) is a method to personalize text2image models like stable diffusion given just a few(3~5) images of a subject.
```

Comment:
```suggestion
[DreamBooth](https://arxiv.org/abs/2208.12242) is a method to personalize text2image models like stable diffusion given just a few (3~5) images of a subject.
```

Reply:
Maybe also link to the LoRA paper https://arxiv.org/abs/2106.09685 and write a few lines about it.